deployment/deploy.sh

```diff
@@ -93,30 +89,26 @@ else
     exit 0
 fi	
 
-#echo "npm publish ..."
-#npm publish
+echo "npm publish ..."
+npm publish
 
 # Reset
 echo "make clean ..."
 make clean
 make
+git commit -am "Resetting to HEAD [ci skip]"
 
-# Cleaning up backup files
-echo "Cleaning up backup files ..."
-rm -f bower.json-e CHANGELOG.md-e package.json-e src/0_config.js-e test/web-config.js-e
-
-#echo "Pushing changes back to repo ..."
-#git push 
-#git push origin $VERSION
+echo "Pushing changes back to repo ..."
+git push origin $CIRCLE_BRANCH
 
 # Send an update to slack channels
 DEPLOY_IMG=http://workshops.lewagon.org/assets/landing-2/deploy-button-5068ec2c575492ba428569111afe3ce6.jpg
 echo "Sending update to slack ..."
-#slackcli -t $SLACK_TOKEN -h int-eng -m "$CIRCLE_USERNAME Deploying WedSDK:$CIRCLE_BRANCH v$VERSION" -u websdk-deploy -i $DEPLOY_IMG
+#uncomment to send updates to int-eng
+#slackcli -t $SLACK_TOKEN -h int-eng -m "$CIRCLE_USERNAME Deployed WedSDK:$CIRCLE_BRANCH v$VERSION" -u websdk-deploy -i $DEPLOY_IMG
```

Comment:
For slack notifications, we should do the same thing we do with other repos for consistency. When we look back at deployment histories, it'll be easier if we do it the same way. The script we use for other repos is in https://github.com/BranchMetrics/deploy/blob/master/scripts/rollbar.py
You can just copy that file in if you don't want to add the whole repo, but do remove the rollbar api keys from the repo and put them into environment variables because this is a public repo.